.gitattributes

```diff
@@ -0,0 +1,5 @@
+/tests export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.travis export-ignore
```

Comment:
wrong file name. The file is named `.travis.yml`

Reply:
@simonschaufi it looks like you have this mistake in other PRs you sent, btw

Reply:
oh yes, right.